backend/app/core/config.py

```diff
@@ -1,21 +1,17 @@
 import secrets
 
-from typing import List
-from pydantic import BaseSettings, AnyHttpUrl, PostgresDsn
+from pydantic import BaseSettings
 
 
 class Settings(BaseSettings):
     SECRET_KEY: str = secrets.token_urlsafe(32)
     ACCESS_TOKEN_EXPIRE_MINUTES: int = 60 * 24
     ALGORITHM = "HS256"
-    USER = "admin"
-    HASHED_PASSWORD = "$2b$12$2jKZIT97V5wCJDDbRD4IpOatUeFYvOBQ2olGSY0TJvWZq1/AsnQnW" #adminnimda
-
+    USER_NAME = "admin"
```

Comment:
to powinno wylądować w repo?

Reply:
Teoretycznie nie, ale na ten moment nie ma gdzie tego wrzucić, można to sobie przenieść do bazy jak będzie trzeba. Produkcja jeszcze nie wystartowała :D